.github/workflows/benchmarks.yml

```
@@ -6,7 +6,7 @@ on:
 
 jobs:
   sdk-benchmarks:
-    runs-on: self-hosted
+    runs-on: github-benchmark-runner
```

TIL you can't target by runner name, only by labels, so I created a new label for this runner...
```suggestion
    runs-on: equinix-bare-metal
```